packages/patternfly-4/react-virtualized-extension/src/components/Virtualized/RowWrapper.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { isEqual } from 'lodash-es';
+import { columnsAreEqual } from 'reactabular-table';
+import { RowWrapper } from '@patternfly/react-table';
+import { VirtualizedBodyContext } from './Body';
+
+class VirtualizedRowWrapper extends React.Component {
+  trRef = null;
+
+  setTrRef = element => {
+    this.trRef = element;
+  };
+
+  updateRowHeight = () => {
+    if (this.trRef) {
+      const { updateHeight, rowProps } = this.props;
+      updateHeight(rowProps['aria-rowindex'], this.getAbsoluteHeight(this.trRef));
+    }
+  };
+
+  // offsetHeight does not include margins, so we use this helper for better accuracy
+  getAbsoluteHeight = el => {
+    const styles = window.getComputedStyle(el);
+    const margin = parseFloat(styles.marginTop) + parseFloat(styles.marginBottom);
+    return Math.ceil(el.offsetHeight + margin);
+  };
+
+  static shouldComponentUpdate(nextProps) {
+    const { columns, rowData } = this.props;
+    // Update only if a row has not been measured and either
+    // columns or rowData hasn't changed
+    if (nextProps.rowData._measured) {
+      return !(columnsAreEqual(columns, nextProps.columns) && isEqual(rowData, nextProps.rowData));
```

Comment:
1. `columnsAreEqual` is not enough because user can have arbitrary data tied to each `column`
2. `isEqual` can be slow because the user can have large structures in `rowData`. 
Isn't there an alternative way ? What are the problems of updating the component every time?

Reply:
removed this method - it was legacy logic from reactabular-virtualized. We can revisit this if perf problems become an issue